src/Cello/simulation_Simulation.cpp

```diff
@@ -726,6 +731,17 @@ void Simulation::initialize_hierarchy_() throw()
 			   config_->mesh_root_blocks[1],
 			   config_->mesh_root_blocks[2]);
 
+  bool lp3[3] = { false, false, false };
```

Comment:
Out of curiosity, were there any problems with the fact that we weren't initializing the periodicity values here in the past? Or did some other change require us to initialize them here?

Reply:
I don't specifically remember, but according to a log entry I "Replaced `Block::periodicity()` with `Hierarchy::get_periodicity()`" which this looks consistent with.